### PR TITLE
Ignore .git and .svn directories

### PR DESCRIPTION
**Pull request by julienfalque**
Alternative to #6 to implement #4.

/cc @keradus

### CODE REVIEW

**Comment by keradus** [comment]
is it a good PR to add support for custom exclusion #5 ?

**Comment by julienfalque** [comment]
I'd rather do it in a dedicated PR.

File: tests/tests.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ run_test() {
     cd "${1}files" || return 3
     output=$(../../../check-trailing-whitespaces)
     status_code="$?"
-    cd ../.. || return 3
+    cd "$OLDPWD" || return 3
```
Comment (keradus) [inline tests/tests.sh]: wow

Comment (julienfalque) [reply tests/tests.sh]: This is equivalent to `cd - > /dev/null`.

**Pull request merged**